torch/_dynamo/convert_frame.py

```diff
@@ -850,7 +853,13 @@ def format_guard_failures() -> str:
                 format_guard_failures(),
                 troubleshooting_url,
             )
-            unimplemented(f"{limit_type} reached")
+            if justknobs_check(
+                "pytorch/compiler:skip_code_recursive_on_cache_limit_hit"
```

Comment:
Do you want to have a flag in config as well, just in case?

Reply:
Added!